docs/source/howto/data.rst

```diff
@@ -260,6 +260,338 @@ Finding data
+    PK    Type    Created
+    ----  ------  ---------------
+     1    Code    26D:21h:45m ago
+
+Remove nodes from a Group
+^^^^^^^^^^^^^^^^^^^^^^^^^
+From the command line interface:
+
+.. code-block:: console
+
+    $ verdi group remove-nodes -G test_group 1
+    Do you really want to remove 1 nodes from Group<test_group>? [y/N]: y
+
+From the python interface:
+
+.. code-block:: ipython
+
+    In [1]: group = Group.get(label='test_group')
+
+    In [2]: group.clear()
+
+Rename Group
+^^^^^^^^^^^^
+From the command line interface:
+
+.. code-block:: console
+
+      $ verdi group relabel test_group old_group
+      Success: Label changed to old_group
+
+From the python interface:
+
+.. code-block:: ipython
+
+    In [1]: group = Group.get(label='old_group')
+
+    In [2]: group.label = "another_group"
+
+
+Delete Group
+^^^^^^^^^^^^
+From the command line interface:
+
+.. code-block:: ipython
+
+      $ verdi group delete another_group
+      Are you sure to delete Group<another_group>? [y/N]: y
+      Success: Group<another_group> deleted.
+
+Copy one group into another
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+This operation will copy the nodes of the source group into the destination group.
+Moreover, if the destination group did not exist before, it will be created automatically.
+
+From the command line interface:
+
+.. code-block:: console
+
+    verdi group copy source_group dest_group
+    Success: Nodes copied from group<source_group> to group<dest_group>
+
+From the python interface:
+
+.. code-block:: ipython
+
+    In [1]: src_group = Group.objects.get(label='source_group')
+
+    In [2]: dest_group = Group(label='destination_group').store()
+
+    In [3]: dest_group.add_nodes(list(src_group.nodes))
+
+
+Organizing data with groups
+---------------------------
+
+In this section, we will study some practical examples of how one can use Groups to improve the management of the data.
+
+Case I: groupping structures with a similar property
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Suppose you have a database of structures for which you have performed a set of simulations.
+Your goal is to continue the study only for the structures for which the bandgap is higher than ``0.08 a.u.``
+To achieve that goal you would first need to make a query that finds such structures and puts them into a group.
+Let's go through the whole process step by step.
+
+1. We start with creating the group ``promising_structures`` in the ``bash`` shell:
+
+.. code-block:: console
+
+    $ verdi group create promising_structures
+    Success: Group created with PK = 8 and name 'promising_structures'
+
+2. Then we do a query that finds the structures which have the property ``bandgap_spin1_au`` higher than  ``0.08``.
+
+.. code-block:: python
+
+    q = QueryBuilder()
+    q.append(StructureData,  tag='structure', project='*') # Here we are projecting the entire structure object
+    q.append(Cp2kCalculation, with_incoming='structure', tag='calculation')
```

Comment:
Can we replace the very specific `Cp2kCalculation` and `bandgap_spin1_au` everywhere for the more generic `CalcJobNode` and `bandgap`, respectively?